doc/For developers/Technical documentation/Docker_HNS.md

```diff
@@ -0,0 +1,54 @@
+## How to run docker daemon in debug mode
+
+Just run dockerd.exe -D in console or add "debug": true to config and restart daemon.
+
+## What exactly happens during container creation
+
+For windows there is only one [graph driver](https://blog.mobyproject.org/where-are-containerds-graph-drivers-145fc9b7255) which is called windowsfilter. It is basically wrapper for hcsshim requests which is wrapper for HCS requests.
+
+Command discussed: docker run -id --name example microsoft/windowsservercore powershell
```

Comment:
What do you think about making structure of this section a bit more clear? E.g.:

`
Command "docker run -id --name example microsoft/windowsservercore powershell" results in sending the following POST request "/v1.39/containers/create?name=example".

It triggers the following actions:
I. Creating container
1.
2. ...
II. Starting container
1.
2. ...
`